modules/007-registrypackages/images/kubernetes-cni-centos/scripts/install

```diff
@@ -14,7 +14,15 @@
 # limitations under the License.
 
 set -Eeo pipefail
-if ! rpm --quiet -q "$(rpm -qip kubernetes-cni.x86_64.rpm | grep Name | awk '{print $3}')"; then
-  rpm -U --nodeps kubernetes-cni.x86_64.rpm
+package="kubernetes-cni"
+package_file="${package}.x86_64.rpm"
+rpm_name="$(rpm -qip ${package_file} 2>/dev/null | grep Name | awk '{print $3}')"
```

Comment:
in the next version we will think about move duplicate code to template and use templates.